pkgs/by-name/br/briar-desktop/package.nix

```diff
@@ -58,6 +59,8 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
This needs a custom update script, because `nix-update` can't check the briar page.